### PR TITLE
Replace PDF Reader

### PR DESCRIPTION
**Pull request by EricBanker12**
# Description

As a Child, I would like to Highlight, so that I can copy or read along with audio

Also, fixes issue of PDF not loading on Firefox and Edge browsers.

[Trello Card](https://trello.com/c/7TMXeIIY/47-child-react-pdf-for-text-highlighting)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Bugfix
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] Tested Component Rendering

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by EricBanker12** [comment]
[[Child] - React-PDF for text highlighting](https://trello.com/c/7TMXeIIY/47-child-react-pdf-for-text-highlighting)

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
         "@types/pdfjs-dist": "^2.1.3",
```
Comment (wSedlacek) [inline package.json]: If you are remove `pdfjs-dist` then you need to remove the types too

File: src/app/components/pdf-display/pdf-display/pdf-display.component.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Document, Page } from 'react-pdf';
```
Comment (wSedlacek) [inline src/app/components/pdf-display/pdf-display/pdf-display.component.tsx]: This is going to have a large bundle size.
Please refer to https://reactjs.org/docs/code-splitting.html

**Review by wSedlacek — CHANGES_REQUESTED** [review]
Let's try to get `react-pdf` out of the main bundle.
If necessary dynamically load all of `pdf-display`

**Review by ghost — COMMENTED** [review]
replace PDF reader.

**Review by wSedlacek — APPROVED** [review]
Screen Shot 2020-02-03 at 12 58 54

The size looks much better if this works in staging then it is good to merge

**Pull request merged**